rgis-events/src/lib.rs

```diff
@@ -121,14 +121,16 @@ pub struct ZoomCameraEvent {
     /// * `amount ∈ [1]` → no change
     /// * `amount ∈ (0, 1)` → zoom out
     pub amount: f32,
+    pub coord: geo::Coord,
```

Comment:
Can you change this to be `geo_projected::Projected<geo::Coord>` to indicate this should be a projected coordinate?